packages/nbdime/typings/codemirror-metamode.d.ts

```diff
@@ -3,21 +3,19 @@
 // Definitions by: jacqt <https://github.com/vidartf>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-
 // Typings from the codemirror/mode/meta module
```

Comment:
These typings are pulled from external sources, so probably best to ignore.

Reply:
@vidartf Ok I can add `typings`directory to the `.prettierignore`file if this is what you mean.